pjnath/include/pjnath/config.h

```diff
@@ -468,6 +468,26 @@
 #endif
 
 
+/**
+ * For trickle ICE, this macro specifies the maximum time of waiting for
+ * end-of-candidates indication from remote once ICE connectivity checks
+ * is started, in seconds. When the timer expires, ICE will assume that
+ * end-of-candidates indication is received so any further remote candidate
+ * update will be ignored.
+ *
+ * Note that without remote end-of-candidates indication, ICE will not be
+ * able to conclude the ICE negotiation. Also note that the ICE connectivity
+ * checks should only be started after both agents have started trickling
+ * ICE candidates (e.g: both have sent their SDPs, either via normal SDP
+ * offer/answer or SIP INFO).
+ *
+ * Default: 60 seconds.
```

Comment:
60 sec seems too long? Is there any recommendation for this?

Reply:
I can't find any recommendation for this.

The total timeout (with retransmissions+response timeout) of a STUN tsx is ~15s, the default DNS query timeout is ~10s, so a STUN tsx may take ~25s. A TURN allocation may require few STUN tsx (e.g: retry next server), so I guess 60s is a fair value (about 1 DNS query + 3 retry)?

Reply:
By 30 sec, half of the period, I imagine most users will have already concluded that there is no media flow and hang up the call.
But this is minor only, app can simply change the default timeout as it wishes.

Reply:
Hmm.. okay, will change to 40 sec (1 DNS query + 2 retry/signalling delay), to give a TURN backup server a better chance, and perhaps to accommodate a few patient users :)

Also updated the docs to clarify about the relation of the end-of-cand indication & ICE negotiation conclusion.

Reply:
Also note that the timer can be started since receiving 180 response (instead of 200), as SIP INFO (with trickle ICE SDP) traffics should be started after dialog is established (response with To tag).